action.yml

```diff
@@ -0,0 +1,141 @@
+name: Plugin Build
+description: Build the plugin and create a zip file.
+
+inputs:
+  PACKAGE_VERSION:
+    required: false
+    description: 'Optional. The version of the package to use.'
+    default: ""
+  DEPLOY_BUILDS_ENDPOINT:
+    required: true
+    description: 'The endpoint to deploy the builds to.'
+    default: ""
+  REPOSITORY_NAME:
+    required: true
+    description: 'The name of the repository.'
+    default: ""
```

Comment:
I see that you have a few required input.vars
In the action call I see that you don't always pass them.
Maybe it is better to make them optional.
E.g, https://github.com/elementor/elementor/pull/28308/files#diff-1b5aac1ec85d5af43f331dd9f3142bdb3481d67809cb8d2bbeca281e40552fd9R35